cli/index.js

```diff
@@ -8,14 +8,14 @@ const regexGuid = /^[0-9a-f]{8}-[0-9a-f]{4}-[1-5][0-9a-f]{3}-[89ab][0-9a-f]{3}-[
 prog
   .version(pjson.version)
   .command('encrypt', 'Encrypt data')
-    .argument('<data>','Data to encrypt')
-    .argument('<service-account>', 'Deployment service account')
-    .argument('<namespace>', 'Deployment namespace')
-    .action(encrypt)
+  .action(encrypt)
+  .option('--secret <data>','Data to encrypt', prog.REQUIRED)
```

Comment:
This makes the arguments global options, and not only for the encrypt commands - correct? Isn't it weird?

Reply:
I don't think it makes the arguments global. I chained another `command` method and didn't see that arguments in the second command help.